# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1474535905 -7200
#      Thu Sep 22 11:18:25 2016 +0200
# Node ID 42f32e5159088ae0abd1dfb36cd8eff0d3119d77
# Parent  027ac2c5d57476c8f1bce96ed83bee404528f38e
de_DE.ts: Update german translation of GUI

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -2821,17 +2821,17 @@ oder wenn unterschiedliche Schriftgrößen oder -stile (fett/kursiv/normal) verwendet werden.</translation>
     <message>
         <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Octave Einstellungen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
-        <translation>Unfähig die Einstellungen zu speichern. Fehlende Datei oder unbekanntes Verzeichnis.</translation>
+        <translation>Die Einstellungen können nicht gespeichert werden.  Fehlende Datei oder unbekanntes Verzeichnis.</translation>
     </message>
     <message>
         <location line="+18"/>
         <location line="+4"/>
         <location line="+444"/>
         <source>System setting</source>
         <translation>Systemeinstellung</translation>
     </message>
